pkg/option/resolver/resolver.go

```diff
@@ -89,6 +89,7 @@ func ResolveConfigurations(ctx context.Context, client client.Clientset, nodeNam
 		}
 	}
 
+	sort.Strings(sourceDescriptions)
```

Comment:
I don't think this should be sorted; the ordering of sources matters and could be relevant while debugging.

Reply:
This is sorted for unit testing reasons. Why does the order matter?

Reply:
Why not sort them in the UT?

Reply:
Good idea, updated to sort them in the unit tests instead.